### ISSUE TITLE
Use fabric and forge or modified version. 

### ISSUE THREAD

**Issue by alfx509** [iiiypuk/minecraft-launcher#18]
Hi
Can I ues this launcher with mod or modified version of minecraft.

**Comment by iiiypuk** [comment]
Yes. See [portable version](https://iiiypuk.github.io/minecraft-launcher/#/portable)
English Guide coming soon.
